ExtractUserDataActivity/handler.ts

```diff
@@ -379,17 +394,45 @@ export const queryAllUserData = (
     .chain(profile =>
       sequenceS(taskEither)({
         // queries all messages for the user
-        messages: fromQueryEither<ReadonlyArray<RetrievedMessageWithContent>>(
-          () => iteratorToArray(messageModel.findMessages(fiscalCode)),
-          "findMessages"
-        ),
+        messages: messageModel
```

Comment:
can we use this one instead: https://github.com/pagopa/io-functions-commons/blob/master/src/models/message.ts#L225 ?